kuma/static/js/wiki.js

```diff
@@ -191,11 +191,26 @@
         Syntax highlighting scripts
     */
     if($('article pre').length && ('querySelectorAll' in doc)) (function() {
-        var syntaxScript = doc.createElement('script');
-        syntaxScript.setAttribute('data-manual', '');
-        syntaxScript.async = 'true';
-        syntaxScript.src = mdn.staticPath + 'js/syntax-prism-min.js?build=' + mdn.build;
-        doc.body.appendChild(syntaxScript);
+        if (mdn.assets && mdn.assets.js.hasOwnProperty('syntax-prism')) {
+            mdn.assets.js['syntax-prism'].forEach(function(url, index, array) {
+                /*
+                   Note: In development, five scripts are loaded, and the
+                   later scripts use Prism, which is declared in the first
+                   script.  This means syntax highlighting often doesn't work
+                   on the first page load. Refresh, and syntax highlighing
+                   should work.
+
+                   To fix this, we'd have to use something like require.js or
+                   in-client merging of JS files.  Maybe django-pipeline will
+                   help as well.
```

Comment:
yes, I tried omitting async as well.  It's up to the browser though, and my test browser continued to load the async scripts first, because they were much smaller and downloaded faster.  I decided to leave out async / no async, to prevent people from thinking it was a solved problem.